pykern/pkconst.py

```diff
@@ -31,3 +33,33 @@
 
 #: Copied from numconv, which copied from RFC1924
 BASE62_CHARS = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'
+TWO_PI = 2 * math.pi
```

Comment:
Document each of these with `#:` prefixes. See two lines above. It's fine for the comment to be simple:

```py
#: 2 * PI
TWO_PI = 2 * math.pi
```

Sort constants alphabetically except if there's a dependency.

All constants are upper case since that's PEP8 compliant.

I would suggest renaming TWO_PI to PI_2, but I know that people may be particular.

Reply:
Changing TWO_PI to PI_2 would require other changes for the sake of being consistent.

More importantly, scientists might wonder what it means and have to double check.
In contrast, TWO_PI will be unambiguous for scientists and mathematicians.

I recommend keeping the name as is.